src/app/components/ConnectorForm/index.tsx

```diff
@@ -27,92 +26,71 @@ function ConnectorForm({
   children,
   video,
   image,
+  logo,
 }: Props) {
-  const { t: tCommon } = useTranslation("common");
-  const navigate = useNavigate();
-
   const media = (
     <div className="flex h-full justify-center items-center">
-      {video ? (
+      {video && (
         <div
-          className="flex-1 relative h-0 max-md:mb-4"
+          className="flex-1 relative h-0 mb-6"
           style={{ paddingBottom: "56.25%" }}
         >
           <video className="absolute t-0 l-0 w-full h-full" controls>
             <source src={video} type="video/mp4" />
           </video>
         </div>
-      ) : (
+      )}
+      {image && (
         <>
-          {image ? (
-            <>
-              <div className="w-full md:w-full mb-8 lg:w-full lg:mb-0">
-                <img
-                  src={image}
-                  alt="Screenshot"
-                  className="block w-full rounded-md"
-                />
-              </div>
-            </>
-          ) : (
-            <>
-              <div className="w-16 md:w-32 mb-8 lg:w-64 lg:mb-0">
-                <img
-                  src="assets/icons/alby_logo.svg"
-                  alt="Alby"
-                  className="block dark:hidden w-full"
-                />
-                <img
-                  src="assets/icons/alby_logo_dark.svg"
-                  alt="Alby"
-                  className="hidden dark:block w-full"
-                />
-              </div>
-            </>
-          )}
+          <div className="w-full mb-6">
+            <img
+              src={image}
+              alt="Screenshot"
+              className="block w-full rounded-md"
+            />
+          </div>
         </>
       )}
     </div>
   );
 
   return (
     <form onSubmit={onSubmit}>
-      <div className="relative lg:flex lg:gap-4 mt-14 bg-white dark:bg-surface-02dp p-10 shadow rounded-lg">
-        <div className="lg:w-1/2">
+      <div className="max-w-[600px] mx-auto relative mt-14 bg-white dark:bg-surface-02dp p-10 shadow rounded-lg">
+        <div className="flex items-center mb-4">
+          {logo && <img src={logo} className="w-16 mr-4 rounded-lg" />}
+          {/*
+              TODO: this can be simplified to always wrap the title in h1, without checking if it
```

Comment:
Good catch, are you able to fix it in this PR? or think it should be done in a follow-up PR?

Reply:
Maybe a new issue that recognizes the complexity of this refactoring need?

Reply:
Sure, thanks for adding a TODO.